script.myepisodes/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.myepisodes"
+       name="MyEpisodes"
+       version="3.0.0"
+       provider-name="Maxime Hadjinlian">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.6.3"/>
+    <import addon="script.module.requests" version="2.22.0"/>
+  </requires>
+  <extension point="xbmc.service" library="default.py"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">MyEpisodes</summary>
+    <description lang="en_GB">Automatically set watched episodes on MyEpisodes.com</description>
+    <platform>all</platform>
+    <language>en fr it pt es</language>
```

Comment:
please drop, this only applies to actual content (media files)

Reply:
What do you mean by this, I'm not sure what you're referring to there?

Reply:
oh okay the language key, is that right?

Reply:
Yes the language element